shared/entities/custom-project.entity.ts

```diff
@@ -78,7 +79,7 @@ export class CustomProject {
 
   @Column({ name: "input_snapshot", type: "jsonb" })
   // TODO: this should be the infered type of the zod schema
-  input: any;
+  input: CreateCustomProjectDto | any
```

Comment:
@alepefe if I remove any from here, your custom project stub used for tests is exploding because of typing, but to be honest I do not understand why. probably related to the whole type fuck up